addons/account/models/account.py

```diff
@@ -464,11 +466,11 @@ def _belong_to_company(self):
     @api.multi
     def _search_company_journals(self, operator, value):
         if value:
-            recs = self.search([('company_id', operator, self.env.user.company_id.id)])
+            recs = self.with_context(active_test=False).search([('company_id', operator, self.env.user.company_id.id)])
         elif operator == '=':
-            recs = self.search([('company_id', '!=', self.env.user.company_id.id)])
+            recs = self.with_context(active_test=False).search([('company_id', '!=', self.env.user.company_id.id)])
         else:
-            recs = self.search([('company_id', operator, self.env.user.company_id.id)])
+            recs = self.with_context(active_test=False).search([('company_id', operator, self.env.user.company_id.id)])
```

Comment:
As said in the specs: "Archived Jornal should not be displayed in Accounting Dashboard + add an "Archived" filter".
So, `with_context(active_test=False)`is useless and should be removed.